crates/astria-composer/src/ds/types.rs

```diff
@@ -0,0 +1,9 @@
+use ethers::types::Transaction as EthersTx;
+
+#[non_exhaustive]
+pub enum RollupTx {
+    EthersTx(EthersTx),
+}
+
+pub(crate) type RollupChainId = String;
+pub type RollupTxExt = (RollupTx, RollupChainId);
```

Comment:
Please only use aliases if they help legibility.

Also this should not be pub

Reply:
Can remove if you think it doesn't